Architect-Coders/MU8
# Issue #53: Persist Comics into DataBase

**What's in:**

Comics persistence and cache strategy for it.

**Testing Done:**

On real device: Xiaomi note 5.

**How it looks:**
**How to test:**
Launch MU8 app, and go to Comics Category, then you can check the internal database with the persisted data.


Use device file explorer:
![image](https://user-images.githubusercontent.com/2092343/73147630-57bcae80-4086-11ea-8a3b-708ab4f632ca.png)

**/data/data/com.architect.coders.mu8/databases**

![image](https://user-images.githubusercontent.com/2092343/73147695-a10cfe00-4086-11ea-95d6-6af5d5602ded.png)

copy the database into your computer, the open with **DB Browser for SQLite** or any other database viewer.

![image](https://user-images.githubusercontent.com/2092343/73147613-3bb90d00-4086-11ea-8f09-426f48591728.png)

**Signed-off-by:**

Alicia Beltran aliceresponde@gmail.com
